workspace/src/app/views/taskViews/transform/TransformRuleEditor.tsx

```diff
@@ -158,8 +158,13 @@ export const TransformRuleEditor = ({
             const pos = nodePositions[node.nodeId];
             if (pos) {
                 node.position = {
-                    x: pos[0],
-                    y: pos[1],
+                    x: pos.x,
+                    y: pos.y,
+                };
+                node.dimension = {
+                    ...node.dimension,
+                    width: pos.width,
+                    height: pos.height,
```

Comment:
Why is the height saved and restored if the user cannot change it? Can this lead to problems in combination with advanced parameters?